### PR TITLE
Refactor/css and javascript

### PR DESCRIPTION
**Pull request by kal-aalrajhi**
# Description
Cleaned up code, added scroll and alt text. Reordered functions, rules, and properties.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring

# How Has This Been Tested?
- [x] open localHost
- [x] dev tools

# Checklist:
- [x] My code follows the Turing's guidelines of this project
- [x] I have performed a self-review of my own code
- [x] No console error messages

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -129,15 +68,79 @@ button:hover {
...
+  margin-top: 20px;
+}
+
+.saved-palettes {
+  border-left: 3px solid black;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  width: 20%;
+  overflow-y: scroll;
```
Comment (whiteheadol) [inline styles.css:113]: I love this feature!

**Comment by laytonmaes** [comment]
looks good to me.

**Comment by whiteheadol** [comment]
I'm so proud of us!

**Comment by laytonmaes** [comment]
wait, please resolve the conflicts, thanks!

**Pull request merged**